### ISSUE TITLE
About los

### ISSUE THREAD

**Issue by hxySherry** [gurkirt/FPN.pytorch1.0#7]

**Comment by hxySherry** [comment]
Why do I train my dataset with a loss of NaN for each iteration?

**Comment by gurkirt** [comment]
Reduce the learning rate and try again.
